src/addrdb.cpp

```diff
@@ -10,6 +10,7 @@
 #include <clientversion.h>
 #include <hash.h>
 #include <random.h>
+#include <stdint.h>
```

Comment:
Nit: Could you `#include <cstdint>` throughout this. The `#include <stdint.h>` form is deprecated (but unlikely to be purged any time soon) :)

Reply:
Sounds good, will make that change. I intentionally used the deprecated version simply because there were 138 instances of stdint.h and only 18 cstdint, but I agree cstdint is the better option. Only downside of cstdint is that we have to tack on std:: to all the uses.

Reply:
Ah neat, I was mistaken, cstdint provides the global-namespace types as well as the std namespace. Just swapping the includes was sufficient.

Reply:
@practicalswift I disagree.
If we want to switch to the non deprecated `<cstdint>` we will need to add something like `using std::uint16_t` as this header doesn't promise injecting the types into global namespace and FWIW the support for C headers is still in C++17 standard and the current draft of C++20 .
See https://github.com/bitcoin/bitcoin/pull/17822#pullrequestreview-344990693